crates/api_crud/src/post/read.rs

```diff
@@ -146,7 +147,9 @@ impl PerformCrud for GetPosts {
       .map(|t| t.local_user.show_read_posts);
 
     let sort: Option<SortType> = from_opt_str_to_opt_enum(&data.sort);
-    let listing_type: Option<ListingType> = from_opt_str_to_opt_enum(&data.type_);
+    let site = blocking(context.pool(), Site::read_local_site).await??;
+    let listing_type: ListingType = from_opt_str_to_opt_enum(&data.type_)
+      .unwrap_or(ListingType::from_str(&site.default_post_listing_type)?);
```

Comment:
Wanna note that this will probably be ignored, as the front end nearly always sends this, and I *think* has its own defaults. I'll add an issue to use the defaults from GetSite